src/main/java/edu/greenblitz/tobyDetermined/commands/rotatingBelly/RotateInToDoor.java

```diff
@@ -0,0 +1,14 @@
+package edu.greenblitz.tobyDetermined.commands.rotatingBelly;
+
+import edu.greenblitz.tobyDetermined.RobotMap;
+import edu.wpi.first.wpilibj2.command.ParallelRaceGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+
+public class RotateInToDoor extends ParallelRaceGroup {
```

Comment:
should be a deadline group like in  RotateOutOfDoor

Reply:
done